apps/chat/src/components/Chat/ConversationSettingsModels.tsx

```diff
@@ -98,12 +112,14 @@ export const ConversationSettingsModel = ({
             displayCountLimit={
               unavailableModelId ? RECENT_MODELS_COUNT - 1 : RECENT_MODELS_COUNT
```

Comment:
you show fake model and need to decrease displayCountLimit
```suggestion
              isPlayback && unavailableModelId ? RECENT_MODELS_COUNT - 1 : RECENT_MODELS_COUNT
```

Reply:
Thank you for the suggestion!
Changed.